production/common/dtd/ldmlSupplemental.dtd

```diff
@@ -216,7 +216,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 
 <!ELEMENT calendarData ( calendar* ) >
 
-<!ELEMENT calendar ( calendarSystem?, eras? ) >
+<!ELEMENT calendar ( calendarSystem?, inheritEras?, eras? ) >
```

Comment:
I am mystified as to how the changes in this file did not get picked up in production for beta2...

Reply:
not sure